spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/BatchNormalization.scala

```diff
@@ -61,6 +61,19 @@ class BatchNormalization[T: ClassTag](
 
   require(nOutput > 0, "output feature map number must be greater than zero")
 
+  private var parallism : Option[Int] = None
+
+  def setParallism(parallism: Option[Int]): Unit = {
```

Comment:
comments for public methods and variables

Reply:
Maybe
```scala
def setParallism(parallism: Int): Unit = {
    this.parallism = Some(parallism)
}
```
is more user friendly. And should it be private?

Reply:
Makes sense, we cannot make it private , it's an exposed interface